sensor/common/clusterentities/store_test.go

```diff
@@ -284,3 +368,321 @@ func (s *ClusterEntitiesStoreTestSuite) TestChangingIPsAndExternalEntities() {
 }
 
 // endregion
+// region container-id history test
+
+func (s *ClusterEntitiesStoreTestSuite) TestMemoryAboutPastContainerIDs() {
+	type whereContainerIDisStored string
+	const (
+		// the container will be found in history
+		history whereContainerIDisStored = "history"
+		// the container will be found in the containerIDMap
+		theMap whereContainerIDisStored = "the-map"
+		// the container will not be found
+		nowhere whereContainerIDisStored = "nowhere"
```

Comment:
[nit] It could be read as `now here` 😆 
```suggestion
		noWhere whereContainerIDisStored = "no-where"
```

Reply:
nowhere is a perfectly valid word 🤔

Reply:
🤦 you are right. I don't know what I was thinking...